docs/support-guide.md

```diff
@@ -0,0 +1,261 @@
+[sig-storage] CSI Volumes [Driver: csi-hostpath] [Testpattern: Dynamic PV (default fs)] provisioning should provision storage with pvc data source [Suite:openshift/conformance/parallel] [Suite:k8s]
+
+ --> Failed flake tests - Statistic from OpenShift CI
+Flakes  Perc   TestName
+101 10.576% [sig-arch][bz-DNS][Late] Alerts alert/KubePodNotReady should not be at or above pending in ns/openshift-dns [Suite:openshift/conformance/parallel]
+67  7.016%  [sig-arch][bz-Routing][Late] Alerts alert/KubePodNotReady should not be at or above pending in ns/openshift-ingress [Suite:openshift/conformance/parallel]
+2   0.386%  [sig-imageregistry] Image registry should redirect on blob pull [Suite:openshift/conformance/parallel]
+32  4.848%  [sig-network][Feature:EgressFirewall] egressFirewall should have no impact outside its namespace [Suite:openshift/conformance/parallel]
+11  2.402%  [sig-network][Feature:EgressFirewall] when using openshift-sdn should ensure egressnetworkpolicy is created [Suite:openshift/conformance/parallel]
+
+ Data Saved to directory './processed/'
+```
+
+> TODO: create the index with a legend with references to the output.
+
+### Understanding the extracted results <a name="review-process-explain"></a>
+
+The data extracted to local storage contains the following files for each plugin:
+
+- `test_${PLUGIN_NAME}_baseline_failures.txt`: List of test failures from the baseline execution
+- `test_${PLUGIN_NAME}_provider_failures.txt`: List of test failures from the execution
+- `test_${PLUGIN_NAME}_provider_filter1-suite.txt`: List of test failures included on suite
+- `test_${PLUGIN_NAME}_provider_filter2-baseline.txt`: List of test failures tests* after applying all filters
+- `test_${PLUGIN_NAME}_provider_suite_full.txt`: List with suite e2e tests
+
+The base directory (`./results-provider-processed`) also contains the **all error messages (stdout and fail summary)** for each failed test. Those errors are saved into individual files onto those sub-directories (for each plugin):
+
+- `failures-baseline/${PLUGIN_NAME}_${INDEX}-failure.txt`: the error summary
+- `failures-baseline/${PLUGIN_NAME}_${INDEX}-systemOut.txt`: the entire stdout of the failed plugin
+
+Considerations:
+
+- `${PLUGIN_NAME}`: currently these plugins names are valid: [`openshift-validated`, `kubernetes-conformance`]
+- `${INDEX}` is the simple index ordered by test name on the list
+
+> TODO: mention the spreadsheet `failures-index.xlsx` created. It can be used to take notes of what was already reviewed by each layer/stakeholder (partner, support, and engineering) reviewing the errors, and also to improve the communication between the parts. For example, the partner could provide that google sheet with notes to check what he already reviewed (as we [mrbraga and robert] do while reviewing, and submitting it). The support can keep that list updated sharing it with engineering, and so on.
+
+Example of files on the extracted directory:
+
```

Comment:
The number in the example below doesn't seem to match the descriptions list above.